modules/admin-ui-frontend/app/scripts/shared/partials/modals/series-details.html

```diff
@@ -390,7 +390,9 @@ <h2 translate="EVENTS.SERIES.DETAILS.HEADER" translate-values="{{titleParams}}">
         <div class="obj list-obj">
           <header translate="EVENTS.SERIES.DETAILS.TABS.TOBIRA"></header>
           <div class="obj-container">
-            <a translate="EVENTS.SERIES.DETAILS.TOBIRA.DIRECT_LINK" href="{{tobiraData.baseURL}}/!s/:{{resourceId}}"></a>
+            <a translate="EVENTS.SERIES.DETAILS.TOBIRA.DIRECT_LINK" href="{{directTobiraLink}}"></a> <a ng-click="copyTobiraDirectLink()" aria-label="{{'EVENTS.SERIES.DETAILS.TOBIRA.COPY_DIRECT_LINK' | translate}}">
```

Comment:
Shouldn't this second `<a` element start in a new line?

Reply:
Probably, yes. :shrug: